dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1051,6 +1051,15 @@ public String toString() {
   /**
    * UFS related properties.
    */
+  public static final PropertyKey UNDERFS_XATTR_CHANGE_ENABLED =
+      Builder.booleanBuilder(Name.UNDERFS_XATTR_CHANGE_ENABLED)
+          .setDefaultValue(true)
+          .setDescription("When enabled, Alluxio allows user to set/get xAttr from/to UFS. "
+              + "There will be different implementation methods in different UFS, "
+              + "implemented like HdfsUFS.setXattr, ObjectUFS.setTagging, or other ways.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
+          .setScope(Scope.SERVER)
```

Comment:
do you need this on the client as well? As client may fallback to UFS?

Reply:
currently, the implementation of fallback will not call the setXattr() directly, only the pageDoraWorker will call it.